config-handlers/SeedJobsConfig.groovy

```diff
@@ -51,8 +51,8 @@ def seedJobConfig(config){
             hudson.plugins.git.GitSCM.createRepoList(
                 source?.remote, 
                 source?.credentialsId
-            ), 
-            source?.branch ? [new hudson.plugins.git.BranchSpec("*/${source?.branch}")] : [], 
+            ),
+                source?.branch ? [new hudson.plugins.git.BranchSpec("*/${source?.branch}")] : (source?.tag ? [new hudson.plugins.git.BranchSpec("${source?.tag}")] : []),
```

Comment:
@yorammi - let us support both tags and branches in the same way: just remove the `*/` .
```source?.branch ? [new hudson.plugins.git.BranchSpec("${source?.branch}")] : [],```

Please also fix the indentation...

Reply:
@odavid , I really wondered why we need the * there anyway... in my Jenkins servers, I usually don't put it when I configure a Jenkinsfile from a branch. So, will it be OK to just remove it without adding the 'tag' option?

Reply:
Yes

Reply:
In that case, I think I'll delete this PR and create a new one from the latest version